docs/README.md

```diff
@@ -2,35 +2,49 @@
 
-**3. Both Client and Server:** Signature will be stripped on client side before sending messages to server, server will make no attempt to verify message signatures. Chat reporting and "Only Show Secure Chat" will not function, and players will be notified that those features are disabled by the mod when trying to use them.
+**2. Only Server:** Clients will still attach signatures when sending messages to the server, but the server will strip them before relaying them to other players. This way chat reporting will not work for any players that join. You can enable the conversion of player to system messages in the config, to prevent players without the mod from seeing them as "Not Secure"
 
-Although the mod can function when present on only one of the sides, it can be configured to demand itself to be installed on another side in order to play. This way you can install it on server and require all clients that join to have this mod in order to have full functionality, but can also leave it server-only if you prefer. Configuration files are located in <u>NoChatReports</u> subfolder of default config folder.
+**3. Both Client and Server:** Signatures will be stripped on the client side before sending messages to the server, which will not attempt to verify message signatures. Chat reporting and "Only Show Secure Chat" will not function, and players will be notified that those features are disabled by the mod when trying to use them.
 
-Additionally, if installed on client, this mod will disable telemetry (similarly to how [No Telemetry](https://modrinth.com/mod/no-telemetry) does).
+Although NoChatReports can function when it is only present on one of either sides, NoChatReports can be configured to demand itself to be installed on the respective other side in order to play. This way you can install it on the server and require all clients that join to have NoChatReports installed, but you can also leave it server-only if you prefer.
 
-To make Velocity compatible with this mod, set `force-key-authentication = false` in `velocity.toml`. Thanks to [MrMelon54](https://github.com/MrMelon54) for pointing this out.
+Additionally, if installed on the client, NoChatReports will disable Telemetry (similar to what [No Telemetry](https://modrinth.com/mod/no-telemetry) does).
+
+To make Velocity compatible with NoChatReports, set `force-key-authentication = false` in `velocity.toml`. Thanks to [MrMelon54](https://github.com/MrMelon54) for pointing this out.
```

Comment:
This seems kinda outdated because there are more instructions server owners need to follow, might as well just refer to the tutorial wiki page.

Reply:
I re-wrote it to just link to [this part of the wiki](https://github.com/Aizistral-Studios/No-Chat-Reports/wiki/Protecting-Server-Players)